src/model_config_utils.cc

```diff
@@ -784,6 +784,31 @@ GetNormalizedModelConfig(
   return Status::Success;
 }
 
+Status
+SetDefaultInstanceCount(
+    ::inference::ModelInstanceGroup* group, const std::string& backend)
+{
+  const int default_cpu_instance_count = 2;
+  if (group->kind() == inference::ModelInstanceGroup::KIND_CPU) {
+    // Backends opt into the default_cpu_instance_count since
+    // some backends don't perform well/have high overhead
+    // when using multiple instances.
+    bool use_default_cpu_instance_count =
+        (backend.compare(kTensorRTBackend) == 0) ||
+        (backend.compare(kTensorFlowBackend) == 0) ||
+        (backend.compare(kOnnxRuntimeBackend) == 0);
```

Comment:
For `std::string`, you can use `==` directly

Reply:
using overloaded operator instead of `compare()`